### ISSUE TITLE
TeXstudio has scrolling issues when used on a second monitor

### ISSUE THREAD

**Issue by HerpDerpongton** [texstudio-org/texstudio#1066]
### Environment
- TeXstudio: 2.12.22
- Qt: Qt Version 5.12.1, compiled with Qt 5.12.1 R
- OS: Windows 10
- TeX distribution: miktex

### Expected behavior
If the cursor is located anywhere within the tabs inf TeXstudio, it is expected that the corresponding scroll bar scrolls, regardless of the monitor the TeXstudio window is located on.

### Actual behavior
If the TeXstudio window is dragged onto a second (external) monitor connected to my tablet, I can only scroll the code tab if the cursor is located in the right half. In the left half, nothing happens instead. In the bottom right corner, the log tab is scrolled instead.

### How to reproduce
Install TeXstudio. Open it on one screen and drag it over to another connected screen. Then the behavior described above should manifest.

### See also
I have posted a question for this problem on [tex.stackexchange.com](https://tex.stackexchange.com/q/541585/82853). I also have the following gifs that illustrate the problem.
Here is a gif of what happens on the tablet screen; everything works as expected:
[normal scrolling](https://i.stack.imgur.com/GMNX7.gif)

Here is what happens when I scroll in the left and right half of the code window: 
[left half, code does not scroll](https://i.stack.imgur.com/fAiQG.gif)
[right half, scrolling works](https://i.stack.imgur.com/wX9EX.gif)

Here is what happens when I try to scroll the lower right half of the code window. Look closely to the bottom right corner of the gif and you can see the scrollbar of the log window moving:
[scrolling in lower right corner of the code window scrolls log window](https://i.stack.imgur.com/RP0Gc.gif)

**Comment by sunderme** [comment]
i assume the displays have different scaling factors ?

**Comment by HerpDerpongton** [comment]
Yes they do. The tablet screen is smaller but has a much higher resolution.
Tablet: 2736 x 1824 pixel, 10 inch x 6.7 inch
External monitor: 1920 x 1080 pixel, 21 inch x 12 inch

**Comment by sunderme** [comment]
I have not been able to reproduce this
Basically, I don't have a 2 monitor set-up, so if some one with a configuration similar can take a look ...

**Comment by HerpDerpongton** [comment]
Maybe this helps: I also have the WSL installed and installed TeXstudio there. The gui runs via Xming. Here, I had no scrolling problems.
Furthermore, I think I remember having a similar problem with another Qt application, but I don't remember which one.

**Comment by sunderme** [comment]
I suspect that your issue is indeed a qt problem. txs does not have code which reacts to wheel activation according to position. that is handled by qt ...

**Comment by HerpDerpongton** [comment]
The programe I described above **was** TeXstudio, but the problem was actually different, namely the "scaling problem" described [here on tex.stackexchange](https://tex.stackexchange.com/q/475606/82853). 
Back then, I seem to have fixed the problem by putting a file _qt.conf_ next to the _texstudio.exe_ with the content

> [Platforms]
> WindowsArguments = dpiawareness=1

which fixed the scaling problem.
Removing this file reactivates the scaling problem, but seems to solve the scrolling problem. Hence, this is really much a bug caused by Qt.

**Comment by sunderme** [comment]
Not reproducible, probably obsolete
